crates/bevy_core/src/serde.rs

```diff
@@ -39,3 +40,32 @@ impl<'de> Visitor<'de> for EntityVisitor {
         Ok(Name::new(v))
     }
 }
+
+impl Serialize for FrameCount {
+    fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        serializer.serialize_str(self.0.to_string().as_str())
```

Comment:
Is there a reason you're using `serialize_str` instead of `serialize_u32` (or tuple strict, etc.)? If so, it's not immediately obvious to me so it might need a comment.

Reply:
no reason, just a whiff on my part -- I haven't had to manually implement a serde trait before. thanks for the catch!